src/modules/pivotTable/PivotTableEngine.js

```diff
@@ -168,7 +158,6 @@ const buildDimensionLookup = (visualization, metadata, headers) => {
         rowHeaders,
         columnHeaders,
         dataHeaders,
-        primaryDimensions,
```

Comment:
Would it be safe to remove the other return values as well? Except for `columns` and `rows` which seems to be used here https://github.com/dhis2/analytics/blob/4417a91e1debc496c2c3bc26c03839901316d213/src/modules/pivotTable/PivotTableEngine.js#L265